src/reducers/index.js

```diff
@@ -145,40 +206,49 @@ export const setUnusedRDF = (state, action) => {
 }
 
 const handlers = {
-  ITEMS_SELECTED: setItemsOrSelections,
+  APPEND_RESOURCE: appendResource,
   CHANGE_SELECTIONS: setItemsOrSelections,
-  PUBLISH_ERROR: setPublishError,
-  PUBLISH_STARTED: clearServerError,
-  LOADING_QA_RESULTS: loadingQaResults,
-  QA_RESULTS_RECEIVED: qaResultsReceived,
-  RETRIEVE_ERROR: setRetrieveError,
-  RETRIEVE_RESOURCE_TEMPLATE_STARTED: clearServerError,
-  SET_BASE_URL: setBaseURL,
+  CLEAR_RETRIEVE_RESOURCE_ERROR: clearRetrieveResourceError,
+  CLEAR_RETRIEVE_RESOURCE_TEMPLATE_ERROR: clearRetrieveResourceTemplateError,
+  CLEAR_RESOURCE_TEMPLATES: clearResourceTemplates,
+  CLEAR_SAVE_RESOURCE_ERROR: clearSaveResourceError,
+  CLEAR_SAVE_RESOURCE_TEMPLATE_ERROR: clearSaveResourceTemplateError,
   CLEAR_SEARCH_RESULTS: clearSearchResults,
-  SET_SEARCH_RESULTS: setSearchResults,
-  SHOW_GROUP_CHOOSER: showGroupChooser,
   CLOSE_GROUP_CHOOSER: closeGroupChooser,
-  SHOW_RESOURCE_TEMPLATE_CHOOSER: showResourceTemplateChooser,
   CLOSE_RESOURCE_TEMPLATE_CHOOSER: closeResourceTemplateChooser,
+  COPY_NEW_RESOURCE: copyResourceToEditor,
+  ITEMS_SELECTED: setItemsOrSelections,
   LANGUAGE_SELECTED: setMyItemsLang,
-  SHOW_RDF_PREVIEW: showRdfPreview,
+  LANGUAGES_RECEIVED: languagesReceived,
+  LOADING_LANGUAGES: loadingLanguages,
```

Comment:
I wonder if we should change these to `RETRIEVE_LANGUAGES_STARTED/FINISHED` for rconsistency.